lisa/tools/iperf3.py

```diff
@@ -222,10 +222,11 @@ def run_as_client_async(
             f"{self.command} {cmd}", shell=True, sudo=True
         )
 
-        # IPerf output emits lines of the following form when it is running
-        # 132.00-133.00 sec   167 MBytes  1.40 Gbits/sec    5    626 KBytes
-        # check if stdout buffers contain "bits/sec" to determine if running
-        process.wait_output("bits/sec")
+        if not output_json and not log_file:
```

Comment:
Add comments to explain this logic. Why it doesn't need to wait, if the two conditions met.

Reply:
udpate